### ISSUE TITLE
`Kubernetes` is deprecating `Docker`: what you need to know 🙅 

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-devops#100]
`Kubernetes` removed support for `Docker` in 2021 ✂️ ⏳ 

TL;DR Docker as an underlying runtime was deprecated in favor of runtimes that use the [Container Runtime Interface (CRI)](https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/) created for Kubernetes. Docker-produced images will continue to work in your cluster with all runtimes, as they always have.

+ [x] https://www.pluralsight.com/resources/blog/cloud/kubernetes-is-deprecating-docker-what-you-need-to-know
+ [x] https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker

What does this _mean_? 
Specifically: does it mean we should reconsider our use of `Docker`? 💭
